pkg/consensus/consensus.go

```diff
@@ -224,6 +227,10 @@ func (s *service) broadcast(p payload.ConsensusPayload) {
 		pr.minerTx = *s.txx.Get(pr.transactionHashes[0]).(*transaction.Transaction)
 	}
 
+	if err := p.(*Payload).Sign(s.dbft.Priv.(*privateKey)); err != nil {
+		s.log.Warn("can't sign consensus payload")
```

Comment:
You should output an error here also, otherwise this message doesn't have much value.